src/server/pps/server/pipeline_controller.go

```diff
@@ -135,8 +137,12 @@ func (a *apiServer) step(pachClient *client.APIClient, pipeline string, keyVer,
 		if op.pipelineInfo.Stopped {
 			return op.setPipelineState(pps.PipelineState_PIPELINE_PAUSED, "")
 		}
-		// default: scale down if standby hasn't propagated to kube RC yet
+		// pipelines in standby shouldn't be able to crash (CRASHING is most like
+		// RUNNING)
```

Comment:
nit: line break here looks weird

Reply:
I actually ended up removing this comment as it's incorrect—crashingPipelineMonitor takes pipelines out of CRASHING, but it doesn't put them into CRASHING. The reason to cancel it here is because if the pipeline is in PAUSED, then a goro to take it out of CRASHING doesn't make sense.

Marking this resolved.